react-app/src/App.js

```diff
@@ -1,26 +1,54 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { Container, StylesProvider } from "@material-ui/core";
+import clsx from "clsx";
+import React, { Component, Fragment } from "react";
+import { Route, Switch } from "react-router-dom";
```

Comment:
For the front-end there is also a recommended import order, just from a code styling perspective. It goes like:
1. Library/module imports ('react', "clsx", "react-router-dom")
2. Your own component and action files (grouped together)
3. Css files